src/lib/db/migrations/00027-admin-community.js

```diff
@@ -0,0 +1,50 @@
+/** @param {import('postgres').Sql<any>} sql */
+export const up = async (sql) => {
+	// Check if we need to migrate for the admin community and persona.
+	const [community1] = await sql`SELECT * FROM communities WHERE community_id = 1`;
+	if (!community1 || community1.name === 'admin') {
+		return;
+	}
+
+	// At this point, we know we need to migrate existing data to the new assumptions.
+	// The "admin" community and its persona are expected to have id 1.
+
+	// Move the existing community -- all references to `communities.community_id` cascade.
+	await sql`
+		UPDATE communities
+		SET community_id = (1 + (SELECT count(*) from communities))
```

Comment:
```suggestion
		SET community_id = (1 + (SELECT max(community_id) from communities))
```
Ran into a bug where I had deleted communities so there were already communities at the count+1

Reply:
nice catch